app/models/like.rb

```diff
@@ -1,3 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :post_id
+  belongs_to :users
```

Comment:
- [ ] Same here, pleaase make the reference through author by adding belongs to `:author` and classname `User`